DHIS2/cloner/dhis2_clone

```diff
@@ -130,6 +138,19 @@ def magenta(txt):
     return '\x1b[35m%s\x1b[0m' % txt
 
 
+def execute_scripts(scripts_folder):
+    scripts = []
+    files = os.listdir(scripts_folder)
+    for fname in files:
+        filename, file_extension = os.path.splitext(fname)
+        if file_extension in ('.sh','.py'):
+            scripts.append(fname)
+
+    for script in scripts:
```

Comment:
To make sure you can know in advance in which order the scripts are going to be run, you can do instead:

```
for script in sorted(scripts):
```

And then name the scripts with a number first (if there is more than one, and you want), to make the order apparent (like in the init.d scripts in the old days!).